src/utils/parse.ts

```diff
@@ -36,5 +36,5 @@ export default function parse(htmlStr: string) {
   const lines = htmlStr.split('\n')
   const breaks = [...Array(lines.length - 1)].map(() => '\n')
 
-  return html(lines, ...breaks)
+  return html(lines as any, ...breaks)
```

Comment:
The type of the first argumet in the latest `htm` is `TemplateStringsArray`. Unfortunately, I could not find a way to cast `lines` as `TemplateStringsArray`, so I used `as any` here.

Reply:
There is no issue in `htm` v3.0.1.